### ISSUE TITLE
Remove usage of native.bind

### ISSUE THREAD

**Issue by bmuschko** [bazelbuild/rules_groovy#41]
From what I understand `native.bind` [should not be used anymore](https://github.com/bazelbuild/bazel/issues/1952). The recommended replacement is `java_import_external`.

**Comment by ekuefler** [comment]
Agreed that I'd like to remove this, but my main concern is that it's likely to break anyone who's relying on the external bindings as defined at https://github.com/bazelbuild/rules_groovy#setup. I think `bind` is the only way to set up a package in `external`.

Breaking this probably isn't a big deal since we're still version 0.0.X, but I'm not really sure what the standard pattern is for defining these sorts of dependencies these days.

**Comment by bmuschko** [comment]
I think the rules should probably use `rules_jvm_external` to define and resolve dependencies but I will ask on the Bazel Slack.
